crypto/crypto_aes.c

```diff
@@ -70,9 +73,9 @@ useaesni(void)
 
 		/* Test cases: key is 0x00010203..., ptext is 0x00112233... */
 		for (i = 0; i < 16; i++)
-			ptext[i] = 0x11 * i;
+			ptext[i] = 0x11 * (i & 0xff);
```

Comment:
I meant `(0x11 * i) & 0xff`.  We're taking the final byte of the multiplication result.